### ISSUE TITLE
NTR: Malformations of cortical development

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4510]
Related to #4382
suggested synonym: neuronal migration disorder
suggested parent: morphological abnormality of the central nervous system

I think this is the same as Abnormality of neuronal migration (https://hpo.jax.org/app/browse/term/HP:0002269)

Renske, can we leave the current primary label? Malformations of cortical development sounds like it could be almost anything, e.g., cyclopia, and so it is not an unambiguous label.

**Comment by pnrobinson** [comment]
@Renske-Oegema can I relabel this term?

**Comment by Renske-Oegema** [comment]
☹ i cannot find it on git hub

Met vriendelijke groet,

Renske Oegema

[Universitair Medisch Centrum Utrecht]

klinisch geneticus | Divisie Laboratoria, Apotheek en Biomedische Genetica, Sectie Klinische Genetica
Universitair Medisch Centrum Utrecht | Kamernummer KC.04.088.2 | Huispostnummer KC.04.082.0 | Postbus 85090 | 3508 AB  UTRECHT
www.umcutrecht.nl<http://www.umcutrecht.nl>
________________________________

De informatie opgenomen in dit bericht kan vertrouwelijk zijn en is uitsluitend bestemd voor de geadresseerde. Indien u dit bericht onterecht ontvangt, wordt u verzocht de inhoud niet te gebruiken en de afzender direct te informeren door het bericht te retourneren. Het Universitair Medisch Centrum Utrecht is een publiekrechtelijke rechtspersoon in de zin van de W.H.W. (Wet Hoger Onderwijs en Wetenschappelijk Onderzoek) en staat geregistreerd bij de Kamer van Koophandel voor Midden-Nederland onder nr. 30244197.


P Denk s.v.p. aan het milieu voor u deze e-mail afdrukt.
Van: Peter Robinson [mailto:notifications@github.com]
Verzonden: zondag 24 februari 2019 22:32
Aan: obophenotype/human-phenotype-ontology
CC: Oegema-2, R.; Mention
Onderwerp: Re: [obophenotype/human-phenotype-ontology] NTR: Malformations of cortical development (#4510)


@Renske-Oegema<https://github.com/Renske-Oegema> can I relabel this term?

**Comment by pnrobinson** [comment]
@Renske-Oegema 
The question is whether
 Abnormality of neuronal migration (https://hpo.jax.org/app/browse/term/HP:0002269)
is the same as "Malformations of cortical development"/"neuronal migration disorder"

I think that "Malformations of cortical development" might be not specific enough to have as a primary label?

**Comment by pnrobinson** [comment]
I think this is correct as is!
